src/items/equipment.js

```diff
@@ -101,14 +113,25 @@ export default class Equipment extends Item {
         console.log("use", this.name);
     }
 
-    // TODO: be able to have multiple different required levels for different skills
     checkRequiredLevel() {
-        let skill = this.skill.toLowerCase();
-        if (skill == "melee") {
-            skill = "attack";
-        }
+        return (
+            calcLevel(characterData.getSkillXp(this.getRequiredCombatSkill())) >=
+            this.requiredLevel
+        );
+    }
 
-        let level = calcLevel(characterData.getSkillXp(skill));
-        return level >= this.requiredLevel;
+    getRequiredCombatSkill() {
```

Comment:
Can you use this function to clean up some of the logic in https://github.com/EthanVieira/rs-clicker/blob/master/src/targets/enemy.js? Particularly ` getDamageLevel()` and `increaseXp()`

Or do you think that's out of the scope of this PR?

Reply:
Also maybe put a print if the default condition happens because under normal circumstances it should never happen

Reply:
Yessir, [updated](https://github.com/EthanVieira/rs-clicker/pull/51/commits/82e4e87e3f5a42b9ade56c8027b12ac6b0bf5ae0)!

Reply:
and [here](https://github.com/EthanVieira/rs-clicker/pull/51/commits/4511f9c2fe3b1749e72e4c7deae355319819377a)